Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1010,10 +1010,17 @@
 #define ENABLE_WIRELESS_PLAYBACK_TARGET_AVAILABILITY_API 1
 #endif
 
-#if !defined(ENABLE_WK_WEB_EXTENSIONS) && (PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
+#if !defined(ENABLE_WK_WEB_EXTENSIONS) && (PLATFORM(MAC) || PLATFORM(MACCATALYST) || PLATFORM(IOS) || PLATFORM(VISION))
 #define ENABLE_WK_WEB_EXTENSIONS 1
 #endif
 
+#if !defined(ENABLE_WK_WEB_EXTENSIONS_ICON_VARIANTS) \
+    && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 150500) \
+    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 180500) \
+    || (PLATFORM(VISION) && __VISION_OS_VERSION_MAX_ALLOWED >= 20500))
+#define ENABLE_WK_WEB_EXTENSIONS_ICON_VARIANTS ENABLE_WK_WEB_EXTENSIONS
+#endif
```

Comment:
Is there a way to generate these definitions?

Reply:
Not in WebKit.